.github/dependabot.yml

```diff
@@ -2,6 +2,16 @@
 
 version: 2
 updates:
+  - package-ecosystem: "maven"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    ignore:
+       # Only allow for patch release upgrades
+      - dependency-name: '*'
+        update-types: ['version-update:semver-major', 'version-update:semver-minor']
+    rebase-strategy: "disabled"
+    open-pull-requests-limit: 5
```

Comment:
I like the conservative 5 here, CI is going to be in demand! :)

Reply:
Yes. If we can't keep regular merged update activity going with a queue this size we're doing something wrong.